src/nvim/ex_getln.c

```diff
@@ -4512,7 +4512,7 @@ static int open_cmdwin(void)
     wp = curwin;
     set_bufref(&bufref, curbuf);
     skip_win_fix_cursor = true;
-    win_goto(old_curwin);
+    win_goto(cmdwin_old_curwin);
```

Comment:
should `cmdwin_old_curwin` be "cleared" after this?

Reply:
It could be for safety, but it doesn't have to be (it should be valid if cmdwin_type != 0, which I added as a doc comment for it)

Reply:
If setting to NULL avoids accidents, seems worth it to me. I can't judge if that's the case though.

Reply:
It definitely doesn't hurt to add. :+1: